frontend/app/templates/components/footer-main.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="ui center aligned container">
+  <div class="ui stackable inverted divided grid">
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Use Badgeyay</strong>
+        <a class="item" href="#" target="_blank">Sitemap</a>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Categories</strong>
```

Comment:
What is the significance of `Categories` here?

Reply:
Here , we will keep the differenet types of Badges :+1:

Reply:
Can you explain what you mean by different categories?

Reply:
@ParthS007 Can you please help me with the link you are referring to, i can't find one :(

Reply:
Here you go @vaibhavsingh97 https://www.conferencebadge.com/demo